### PR TITLE
seting up trailwind-css 

### PR DESCRIPTION
**Pull request by Stevenkwizera06**
setting up trailwind-css
How to test it manually 
-clone the repo
- run npm run dev:start in your teminal
-copy http://localhost:3000/
-then go to your browser and paste it there and see how trialwind css looks

### CODE REVIEW

**Review by shemalucien — CHANGES_REQUESTED** [review]
remove commented codes in App.css, otherwise good work @Stevenkwizera06

File: package.json [hunk 1]
```diff
@@ -18,13 +18,14 @@
...
     "redux": "^4.1.2"
   },
   "devDependencies": {
     "@babel/core": "^7.17.9",
     "@babel/preset-env": "^7.16.11",
     "@babel/preset-react": "^7.16.7",
+    "autoprefixer": "^10.4.4",
     "babel-jest": "^24.9.0",
     "babel-loader": "^8.2.4",
     "css-loader": "^6.7.1",
```
Comment (mukunzidd) [inline package.json:31]: Install postcss-loader in devDeps

File: postcss.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (mukunzidd) [inline postcss.config.js:3]: you need to import tailwind-css and pass it the config like this.

```
const tailwindcss = require("tailwindcss");

module.exports = {
  plugins: [tailwindcss("./tailwind.config.js"), require("autoprefixer")],
};
```

**Review by mukunzidd — CHANGES_REQUESTED** [review]
In your webpack config add "postcss-loader" after css-loader on line 24. You'll be good to go.

File: tailwind.config.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  content: [
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
```
Comment (mukunzidd) [inline tailwind.config.js]: @Stevenkwizera06 What does this do?

This should be part of the purge option which you can set like this: 

```
purge: {
    enabled: true,
    content: ["./src/**/*.{js,jsx,ts,tsx}"],
  },
```

Guys make sure that you understand `code` you're seeing online. Don't just copy and paste.

**Review by songa1 — APPROVED** [review]
Working fine now, but you did not follow best practices for git naming. This is a very needed PR, so we will not make it a big deal, but next time, make sure you follow best practices from [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).

**Review by shemalucien — APPROVED** [review]

**Review by mukunzidd — APPROVED** [review]
- I'll approve but the Description of the PR is sloppy and incomplete
- you also need to rename the branch.

**Review by arthurjunior250 — APPROVED** [review]

**Pull request merged**